typescript/src/task_operations.ts

```diff
@@ -0,0 +1,176 @@
+import * as util from 'util';
+import * as moment from 'moment';
+import { forEachProject, splitFirstSpace } from "./helper";
+import { println } from "./readlinehelper";
+import { Task, TaskProps } from './task';
+
+
+export class TaskOperations {
```

Comment:
can you add a interface for this class, with all the method

Reply:
segregate the functions as much as possible in different classes  with individual interfaces